app/menu/main-menu.js

```diff
@@ -0,0 +1,76 @@
+const {Menu} = require('electron');
+const electron = require('electron');
+const app = electron.app;
+
+const template = [
+  {
+    label: 'Edit',
+    submenu: [
+      {
+        role: 'undo'
+      },
+      {
+        role: 'redo'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'cut'
+      },
+      {
+        role: 'copy'
+      },
+      {
+        role: 'paste'
+      },
+      {
+        role: 'delete'
+      },
+      {
+        role: 'selectall'
```

Comment:
I feel like these options are supposed to be there for text editing in every app that has text boxes.

I checked the apple HIG [here](https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/OSXHIGuidelines/MenuBarMenus.html#//apple_ref/doc/uid/20000957-CH29-SW7) and it confirms:

> Even if your app doesn’t handle text editing within its documents, the expected Edit commands should be available for use in dialogs and wherever users can edit text.

I haven't checked the Windows or GNOME/KDE guidelines, though. Might be different there.